maps/templates/ssv_rostock.dmm

```diff
@@ -1164,9 +1164,55 @@
 	},
 /turf/open/floor/strata/fake_wood,
 /area/rostock/security/brig_holding_area)
+"adN" = (
+/obj/structure/machinery/camera/autoname/almayer{
+	name = "rostock camera";
+	network = list("Rostock");
+	dir = 8
+	},
+/turf/open/floor/strata/multi_tiles/west,
+/area/rostock/lower_deck/m_hallway)
+"adO" = (
+/obj/structure/machinery/light{
+	dir = 8
+	},
+/turf/open/floor/strata/green3,
+/area/rostock/req)
+"adP" = (
+/obj/structure/pipes/standard/simple/hidden/supply{
+	dir = 5
+	},
+/turf/open/floor/plating/plating_catwalk/strata,
+/area/rostock/req)
+"adQ" = (
+/obj/structure/pipes/standard/simple/hidden/supply{
+	dir = 10
+	},
+/turf/open/floor/strata/multi_tiles/southeast,
+/area/rostock/req)
 "adR" = (
 /turf/open/floor/strata/red4/north,
 /area/rostock/security/headquarters_bunk)
+"adS" = (
+/obj/structure/machinery/computer/supply/asrs/upp{
+	req_access = list(232)
```

Comment:
shouldnt the UPP req computer just have this inherently-- or in other words, couldn't this be defined on the computer in code rather than map edited